### PR TITLE
v 1.1

### PR DESCRIPTION
**Pull request by alex-prus**
Update 4 and 5 tasks

### CODE REVIEW

File: Task4/src/bsu/prus/tat/Main.java [hunk 1]
```diff
@@ -4,18 +4,20 @@
 import java.util.Scanner;
 
 /**
- * Java homework
- * Task #4
  * Finds the real roots of a quadratic equation
  * @author Alexander Prus
+ * @version 1.1
  * @since 2.10.16
  */
```
Comment (AlexanderSoroka) [inline Task4/src/bsu/prus/tat/Main.java]: Useless blank line

File: Task4/src/bsu/prus/tat/Main.java [hunk 2]
```diff
@@ -4,18 +4,20 @@
...
 
 public class Main {
+    /**
+     * @param args not used
```
Comment (AlexanderSoroka) [inline Task4/src/bsu/prus/tat/Main.java]: Missed brief

File: Task4/src/bsu/prus/tat/Quadratic.java [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
- * Created by alex_prus on 2.10.16.
+ * class - quadratic equation with three coefficients and discriminant
+ * @author Alexander Prus
+ * @version 1.1
+ * @since 2.10.16
  */
 
 public class Quadratic {
-    static final char aName = 'a';
+    static final char aName = 'a'; //names of coefficients
```
Comment (AlexanderSoroka) [inline Task4/src/bsu/prus/tat/Quadratic.java]: use explicit access level specifier

Comment (alex-prus) [inline Task4/src/bsu/prus/tat/Quadratic.java]: Did you mean to add getter of that constants?

**Pull request closed without merging**